fpga/src/main/scala/nexysvideo/HarnessBinders.scala

```diff
@@ -36,3 +46,55 @@ class WithNexysVideoDDRTL extends HarnessBinder({
     ddrClientBundle <> port.io
   }
 })
+
+// Uses PMOD JA/JB
+class WithNexysVideoSerialTLToGPIO extends HarnessBinder({
```

Comment:
Are these GPIO Pins or FMC pins?

Reply:
pmod, I saw Arty one used pmod, so I mapped these to nexysvideo's pmod JA/JB pins.

Reply:
I see. 

Can you also push a version that maps it to FMC?